config/more_samples-glyf_colr_1.py

```diff
@@ -219,7 +223,7 @@ def _gradient_p2_skewed(accessor_char):
     return SampleGlyph(
         glyph_name=glyph_name,
         accessor=accessor_char,
-        advance=_UPEM,
+        advance=1250,
```

Comment:
Suggest adding a _ADVANCE const

Reply:
...in reading further I see it's not consistent. Maybe just a comment ot explain why it isn't _UPEM?